content/en/tests/early_flake_detection.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Early Flake Detection
+kind: documentation
+description: Detect flakiness before it reaches the default branch
+aliases:
+- /continuous_integration/tests/early_flake_detection/
+further_reading:
+  - link: "/tests"
+    tag: "Documentation"
+    text: "Learn about Test Visibility"
+  - link: "/quality_gates"
+    tag: "Documentation"
+    text: "Learn about Quality Gates"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Early Flake Detection is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+{{< callout url="#" btn_hidden="true" >}}
+Early Flake Detection is in public beta.
+{{< /callout >}}
+
+## Overview
+
+Early Flake Detection is Datadog's mechanism to tackle test flakiness. By identifying tests that are newly added and running them multiple times, Datadog can detect flakiness before it reaches the default branch. A study shows that up to [75% of flaky tests][1] can be identified this way.
+
+## How it works
+
+* Datadog's backend automatically stores unique tests for a given test service. These are **known tests**.
+* Before a test session runs, the Datadog library fetches the list of known tests.
+* If a test is in this list, it's executed normally.
+* If a test is **not** in this list, the Datadog library automatically retries it up to 10 times.
```

Comment:
I would just re-iterate that not being on the list makes it "new".

If a test is **not** in this list, the test is considered **new**, and the Datadog library will automatically retry the test up to 10 times.